api/src/main/java/jakarta/servlet/SessionCookieConfig.java

```diff
@@ -236,4 +238,44 @@
      * @see jakarta.servlet.http.Cookie#getMaxAge
      */
     public int getMaxAge();
+
+    /**
+     * Sets the SameSite attribute for the session tracking cookies created on behalf of the application represented
+     * by the <tt>ServletContext</tt> from which this <tt>SessionCookieConfig</tt> was acquired.
+     * When <code>sameSite</code> is not <code>null</code>, this cookie will have its <code>SameSite</code>
+     * attribute set with the provided value. When <code>sameSite</code> is <code>null</code>, the container may
+     * use a context specific default for the cookie's <code>SameSite</code> attribute or it may not be set.
+     *
+     * @param sameSite the <i>SameSite</i> attribute of the session tracking cookies created on behalf of the
+     *                 application represented by the <tt>ServletContext</tt> from which this
+     *                 <tt>SessionCookieConfig</tt> was acquired. May be null to imply that the SameSite attribute
+     *                 should be set to a context specific default value, or should not be set if there is no context
+     *                 default.
+     *
+     * @throws IllegalStateException if the <tt>ServletContext</tt> from which this <tt>SessionCookieConfig</tt> was
+     *                               acquired has already been initialized
+     *
+     * @see jakarta.servlet.http.Cookie#setSameSite(SameSite)
+     *
+     * @since Servlet 5.0
```

Comment:
Needs to be 5.1 now

Reply:
And replace `<tt>` by `<code>` or `{@code}`. The `<tt>` is disallowed since the HTML5 era.